fe/fe-core/src/main/java/com/starrocks/catalog/TabletInvertedIndex.java

```diff
@@ -569,7 +569,6 @@ public void deleteReplica(long tabletId, long backendId) {
     public Replica getReplica(long tabletId, long backendId) {
         readLock();
         try {
-            Preconditions.checkState(tabletMetaMap.containsKey(tabletId), tabletId);
```

Comment:
why remove this?

Reply:
remove this assert and leave it to the caller to check nullable.